src/configuration/app-containers.md

```diff
@@ -254,35 +255,35 @@ web:
             ...
 ```
 
-It has a few subkeys, which are:
+It has a few subkeys listed below:
 
 * `root`:
-    The folder from which to serve static assets for this location,
-    relative to the application root. The application root is the directory in
-    which the `.platform.app.yaml` file is located.  Typical values for this
+    The folder from which to serve static assets for this location
+    relative to the application root. The application root is the directory
+    where the `.platform.app.yaml` file is located.  Typical values for this
     property include `public` or `web`.  Setting it to `""` is not recommended,
     and its behavior may vary depending on the type of application.  Absolute
     paths are not supported.
 * `passthru`:
     Whether to forward disallowed and missing resources from this location to
-    the application. Can be true, false or an absolute URI path (with leading
-    `/`), the default value is false. For non-PHP applications it will
+    the application and can be true, false or an absolute URI path (with leading
```

Comment:
While we're here, true and false should get backticks around them as they're literal code values.